rules/falco_rules.yaml

```diff
@@ -885,7 +885,7 @@
     (proc.name = calico-node and fd.name startswith /etc/calico)
 
 - macro: prometheus_conf_writing_conf
-  condition: (proc.name=prometheus-conf and fd.directory=/etc/prometheus/config_out)
+  condition: proc.name=prometheus-conf and fd.name startswith /etc/prometheus/config_out
```

Comment:
Actually could you keep the parentheses? We're inconsistent with this but try to use them for any macro so they can be safely embedded without potential confusion with logical operators.

Reply:
Fixed!